scripts/ensure_snap_builds.py

```diff
@@ -83,27 +83,50 @@ def ensure_lp_recipe(
     lp_owner = lp.people[LP_OWNER]
     lp_repo = lp.git_repositories.getDefaultRepository(target=lp_project)
     lp_ref = lp_repo.getRefByPath(path=flavor_branch)
+    lp_archive = lp.archives.getByReference(reference="ubuntu")
+    lp_snappy_series = lp.snappy_serieses.getByName(name="16")
     manifest = dict(
         auto_build=True,
+        auto_build_archive=lp_archive,
         auto_build_pocket="Updates",
         auto_build_channels={"snapcraft": "8.x/stable"},
         description=f"Recipe for {SNAP_NAME} {flavor_branch}",
         git_ref=lp_ref,
-        git_repository=lp_repo,
         information_type="Public",
         name=recipe_name,
         owner=lp_owner,
+        processors=[
+            "/+processors/amd64",
+            "/+processors/arm64",
+        ],
```

Comment:
Similar to the `auto_build_channels`, i think this can be derived from the source branch at some point